smart_simulation/database/db_utils.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import pathlib
+import sqlite3
+
+import pandas as pd
+
+import daiquiri
+from smart_simulation.cfg_templates.config import package_dir
+
+daiquiri.setup(
+    level=logging.INFO,
+    outputs=(
+        daiquiri.output.Stream(
+            formatter=daiquiri.formatter.ColorFormatter(
+                fmt="%(asctime)s [%(levelname)s] %(name)s.%(" "funcName)s: %(message)s"
+            )
+        ),
+    ),
+)
+
+PACKAGE_PATH = pathlib.Path(package_dir)
+LOCAL_PROD_DB = PACKAGE_PATH / "smart_simulation" / "database" / "production.db"
+
+
+def connect(database: pathlib.PurePath = LOCAL_PROD_DB) -> sqlite3.connect:
```

Comment:
Nitpick: I think this returns a `sqlite3.Connection` object, not `sqlite3.connect`. https://docs.python.org/2/library/sqlite3.html#sqlite3.Connection

Reply:
Yes, that's correct. I will make the change.